src/App.css

```diff
@@ -1,7 +1,14 @@
+
+body {
+  background-image: url("/Wittenberg_crop.jpeg");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  background-color: #cccccc; 
+  height: 100vh;
```

Comment:
This is a good solution to setting the background-image. I remember doing something similar on my final project. Sometimes vh can be a little tricky to work with, so keep an eye on that in the future, but it seems implemented well here.